src/sentry/api/event_search.py

```diff
@@ -1743,6 +1743,16 @@ def validate_argument_count(self, field, arguments):
             aggregate=["count", None, None],
             result_type="integer",
         ),
+        Function(
+            "count_geq",
```

Comment:
Are we planning on exposing this to customers as well? We might need a more clear name if we do.

Reply:
That is a clearer name, but really long. I don't have productive suggestions for another name though.

Reply:
Changed the name to `count_at_least`